addon/components/ember-table.js

```diff
@@ -75,6 +75,13 @@ StyleBindingsMixin, ResizeHandlerMixin, {
   // through ctrl/cmd-click or shift-click).
   selectionMode: 'single',
 
+  // A flag to control whether the height of the container is used to
```

Comment:
This should be a multi-line documentation comment. JSDoc only supports comments of exactly the form:
```
/** 
comment text
*/
```